### PR TITLE
Add Chainstack v2.2 release notes

### PR DESCRIPTION
**Pull request by neewy**
- Private hosting
- AWS US East region
- Identity issuers

### CODE REVIEW

File: docs/release-notes.md [hunk 1]
```diff
@@ -8,6 +8,17 @@ meta:
...
 # Release notes
 
+## Chainstack 2.2
+
+*January 19, 2021*
+
+### What's new
+
+* **Private hosting**. Connect your Chainstack organization to Amazon Elastic Kubernetes Service (EKS), and use your EKS clusters to host your blockchain nodes and networks. 
+* **AWS US East region**. You can now deploy your nodes and networks in the <a href="https://docs.aws.amazon.com/AWSEC2/latest/UserGuide/using-regions-availability-zones.html#concepts-available-regions" target="_blank">AWS US East, Northern Virginia</a> region.
```
Comment (akegaviar) [inline docs/release-notes.md:18]: Please spell out the full name for AWS.

Comment (akegaviar) [inline docs/release-notes.md:18]: Links to non-Chainstack resources must be in markdown format.

**Review by CaJIbHuK — APPROVED** [review]

File: docs/release-notes.md [hunk 2]
```diff
@@ -8,6 +8,20 @@ meta:
 
 # Release notes
 
+## Chainstack 2.2
+
+*January 25, 2021*
```
Comment (easeev) [inline docs/release-notes.md:13]: Apparently

```suggestion
*January 22, 2021*
```

File: docs/release-notes.md [hunk 3]
```diff
@@ -8,6 +8,20 @@ meta:
...
+
+### What's new
+
+* **Private hosting**. Connect your Chainstack organization to Amazon Elastic Kubernetes Service (EKS), and use your EKS clusters to host your blockchain nodes and networks.
```
Comment (easeev) [inline docs/release-notes.md:17]: "Connect your Chainstack organization to Amazon Elastic Kubernetes Service" sounds weird

```suggestion
* **Private hosting**. You can now your own Amazon Elastic Kubernetes Service (EKS) infrastructure to deploy blockchain nodes and networks on Chainstack.
```

**Review by easeev — CHANGES_REQUESTED** [review]
Minor changes required

**Review by easeev — APPROVED** [review]
LGTM

**Review by akegaviar — APPROVED** [review]

**Pull request merged**